### ISSUE TITLE
TYPO3 v11+ needs rewrites to work with Symfony Local Web Server

### ISSUE THREAD

**Issue by YetiCGN** [symfony-cli/symfony-cli#286]
I tried to install TYPO3 11.4 LTS and running it with the Local Web Server, as I've done with version 10 hundreds of times before.

Due to [an issue](https://forge.typo3.org/issues/94465) with how they [changed their backend URLs](https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/11.0/Breaking-93048-BackendURLRewrites.html), I can't access a working installation of TYPO3 11 with the Local Web Server. The PHP Development Server however, works perfectly fine.

Output of Symfony Local Web Server:

    [Web Server ] Mar  6 11:11:06 |INFO   | SERVER POST (200) /typo3/install.php host="127.0.0.1:8004" ip="::1" scheme="https"
    [Web Server ] Mar  6 11:11:08 |WARN   | SERVER GET  (404) /typo3/login ip="::1"

Output of PHP Development Server:

    $ php8.1 -S localhost:8000 -t public/
    [Mon Mar  6 11:29:06 2023] PHP 8.1.16 Development Server (http://localhost:8000) started
    [Mon Mar  6 11:29:12 2023] 127.0.0.1:64490 Accepted
    [Mon Mar  6 11:29:15 2023] 127.0.0.1:64490 [200]: GET /typo3/login?loginProvider=1433416747
    [Mon Mar  6 11:29:15 2023] 127.0.0.1:64490 Closing

Basically it comes down to two front controllers, one in `public/index.php` (frontend) and the other in `public/typo3/index.php` (backend). I think what happens is that the URLs get rewritten to use the frontend front controller and the URL `login` is not found in the frontend.

**Comment by tucksaun** [comment]
#259 seems indeed the proper fix for your situation in the long run.
In the meantime, if you want to check that your installation works properly you can try accessing `/typo3/index.php/login` and see if everything else works.
I don't know how typo3 will behave with this and if it will generate proper URLs considering this, but most PHP frameworks I know are usually able to work with both kinds of URLs so you can try 🤞🏻

**Comment by YetiCGN** [comment]
Instead of compiling our own build of the Symfony CLI tool, we've patched the TYPO3 front controller in our dev environments with this package: https://packagist.org/packages/explicatis/typo3-front-controller-redirect

In case anybody comes across this problem. Would be nice to see #259 merged, though.
